gcloud/bigtable/happybase/connection.py

```diff
@@ -174,3 +178,30 @@ def _handle_legacy_args(arguments_dict):
         if arguments_dict:
             unexpected_names = arguments_dict.keys()
             raise TypeError('Received unexpected arguments', unexpected_names)
+
+    def open(self):
+        """Open the underlying transport to Cloud Bigtable.
+
+        This method opens the underlying HTTP/2 gRPC connection using a
+        :class:`.Client` bound to the :class:`.Cluster` owned by
+        this connection.
+        """
+        self._cluster._client.start()
+
+    def close(self):
+        """Close the underlying transport to Cloud Bigtable.
+
+        This method closes the underlying HTTP/2 gRPC connection using a
+        :class:`.Client` bound to the :class:`.Cluster` owned by
+        this connection.
+        """
+        self._cluster._client.stop()
+
+    def __del__(self):
+        try:
```

Comment:
Actually, isn't the `_initialized` flag redundant, given the `_cluster` attribute?  If it was set to `None` at the class level, then you could make `__del__` as:

```python
    def __del__(self):
        if self._cluster is not None:
            self.close()
```